db/migrate/20200225223106_create_circulation_rules.rb

```diff
@@ -0,0 +1,15 @@
+class CreateCirculationRules < ActiveRecord::Migration[5.2]
+  def change
+    create_table :circulation_rules do |t|
+      t.string :short_code
+      t.string :name
+
+      t.timestamps null: false
+    end
+
+    # CirculationRule
+    seed_file = Rails.root.join('db', 'migrate/202002', "#{CirculationRule.name.underscore}.yml")
```

Comment:
Minor/Nitpick, I'd probably push this stuff to a "db/seed" instead of doing it in migration file 🤔

Reply:
That's actually where I started with this.  https://github.com/ualbertalib/discovery/pull/1742#issuecomment-533246127 pointed out that adding this info to an existing production database was most easily done in a migration.  One of the main reasons for doing this is so that we don't have to maintain this information within the discovery repo. If it's a seed then the expectation would be that it should be kept up.

Reply:
Ah makes sense 👍 

I guess my only concern is if a new developer joins this project and they need to create/migrate their local DB or anytime you blow away your current DB and start fresh (which I personally do every time I seed/migrate my DB which happens quite often) you will run these migrations which will seed the DB. Which means you technically have to keep this up to date as well. As you will have to keep `db/migrate/202002/circulation_rule.yml` file in this repo forever and this seeding script in working condition or else this migration will fail.

Maybe why it's rare to see complicated seeding or other scripts in migration files as you want to keep migrations for simply schema changes or change existing data.

If it's a one time thing that we want for production, then maybe that should be a temporary rake task which gets run once in production to seed the new tables. Then afterwards we can remove these files and the rake task and never have to think about this stuff again.

But I'll leave that to you to make the best decision 😄

Reply:
I suppose the other option is leave these how they are. After they get ran in production, come back and cleanup all the scripts for seeding in these migrations and remove the yaml files? This could work too and removes my concerns. Plus could make Unix team job easier

Reply:
I'm going to leave these how they are but I did open a ticket to come back to this https://github.com/ualbertalib/discovery/issues/1932